aredis/client.py

```diff
@@ -261,7 +261,7 @@ def __init__(self, host=None, port=None, startup_nodes=None, max_connections=32,
         self.response_callbacks = self.__class__.RESPONSE_CALLBACKS.copy()
 
     @classmethod
-    def from_url(cls, url, db=None, skip_full_coverage_check=False, **kwargs):
+    def from_url(cls, url, db=None, **kwargs):
```

Comment:
The function sign should be like before or there will be compatibility problem

Reply:
Done.